recipe/meta.yaml

```diff
@@ -1,25 +1,34 @@
-{% set version = "24.04.0" %}
-{% set number = 1 %}
+{% set version = "24.08.0" %}
+{% set number = 2 %}
+
+{% set major_version = version.split(".")[0] %}
+{% set minor_version = version.split(".")[1] %}
 
 package:
   name: mathdx
   version: {{ version }}
 
 source:
-  - url: https://files.pythonhosted.org/packages/py3/n/nvidia-mathdx/nvidia_mathdx-24.4.0-py3-none-any.whl
-    sha256: 5be02c90487816c9e4f5aceb13f1466edb7e2c86608bf04eec5079e148667179
+  - url: https://developer.download.nvidia.com/compute/cuFFTDx/redist/cuFFTDx/nvidia-mathdx-{{ version }}.tar.gz
+    sha256: 66e4a0a1d15969c6b099e15e6f89a89a378b630b8ec7dca18f3e573bf403e0a2
 
 build:
   number: {{ number }}
   skip: true  # [not (linux64 or aarch64)]
```

Comment:
Since this is a header-only library. It can be declared `noarch: generic` and depend on the `__unix` package at runtime. Then we don't have to build separately for arm and x86.

https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-packages-with-os-specific-dependencies